src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -119,9 +125,19 @@ public final void setExcludedClasses(String... excludedClasses) {
             Collections.unmodifiableSet(Arrays.stream(excludedClasses).collect(Collectors.toSet()));
     }
 
+    /**
+     * Sets user-excluded pakcages to ignore. All exlcuded packages should end with a period,
+     * so it also appends a dot to a package name.
+     * @param excludedPackages the list of packages to ignore.
+     */
+    public final void setExcludedPackages(String... excludedPackages) {
+        this.excludedPackages = Collections.unmodifiableSet(Arrays.stream(excludedPackages)
+            .map(x -> x + DOT).collect(Collectors.toSet()));
```

Comment:
user should provide packge names without dots as package name is identifier without trailing dot.